jenkins/databricks/test.sh

```diff
@@ -90,6 +91,18 @@ run_pyarrow_tests() {
 
 ## Separate the integration tests into "CI_PART1" and "CI_PART2", run each part in parallel on separate Databricks clusters to speed up the testing process.
 if [[ $TEST_MODE == "DEFAULT" || $TEST_MODE == "CI_PART1" ]]; then
+    # Run two-shim smoke test with the base Spark build
+    if [[ "$WITH_DEFAULT_UPSTREAM_SHIM" != "0" ]]; then
+        if [[ ! -d $HOME/spark-3.2.0-bin-hadoop3.2 ]]; then
+            wget https://archive.apache.org/dist/spark/spark-3.2.0/spark-3.2.0-bin-hadoop3.2.tgz -P /tmp
+            tar xf /tmp/spark-3.2.0-bin-hadoop3.2.tgz -C $HOME
+            rm -f /tmp/spark-3.2.0-bin-hadoop3.2.tgz
+        fi
+        SPARK_HOME=$HOME/spark-3.2.0-bin-hadoop3.2 \
```

Comment:
Is 3.2.0 base spark mandatory? 

This changes SPARK_HOME from DB spark runtime to the 3.2.0 base spark

We should change SPARK_HOME back to DB runtime after the SMOKE test, to make sure the remaining tests are against DB spark runtime.

Reply:
> Is 3.2.0 base spark mandatory?

Yes, the test with a multi-shim Plugin including the DBR support but running in a non-DBR Spark runtime is exactly what is missing in the pre-merge pipeline. And we also made it hard #11428


> We should change SPARK_HOME back to DB runtime after the SMOKE test, to make sure the remaining tests are against DB spark runtime.

 Note the `\` for the line continuation. The scope of the environment on L101-103 is just the command on L104.